src/hammer-vlsi/defaults.yml

```diff
@@ -206,3 +205,10 @@ par.inputs:
   # the final GDS.
   # type: bool
   gds_merge: false
+
+# The submit command to use. "none", "bare", or null will run on the current host. See hammer_submit_command.py for other options.
+synthesis.submit_command: "none"
+synthesis.submit_command_settings: null
```

Comment:
A few thoughts.
* Would recommend creating a struct to hold these (see how it's done for ExtraLibrary for example). It will make it easier to check these. The main question is how we store commands...
* Would also put these under their respective tool settings (e.g. for this one, put it under `synthesis.`)
* Perhaps it would even make sense to do a default submit_command have each tool override it?

Reply:
I agree with the default plus override per tool.

Reply:
I'm thinking something like this:
```
synthesis.submit:
  # Submission method to use.
  # "direct"/"bare"/"none" will 
  # TODO: make this point at some default submit
  # type: str
  command: "direct"

  # Options for the submission tool.
  # Specified as a list of dictionaries that will be combined.
  # type: List[Dict[str, Any]]
  options: []
```

And then the submission tool will consume this list of dictionaries, squash it, and then do stuff with it. I'm envisioning each submission tool having its own namedtuple/struct-like class with a from_setting class that looks something like https://github.com/ucb-bar/hammer/blob/master/src/hammer-vlsi/hammer_vlsi/hammer_tool.py#L59

Reply:
Fixed